baxter_collaboration_lib/test/test_gripper.cpp

```diff
@@ -0,0 +1,38 @@
+#include <gtest/gtest.h>
+
+#include "robot_interface/gripper.h"
+
+using namespace std;
+
+TEST(GripperTest, testConstructorDefaultValues)
+{
+	string 	limb = "left";
+	bool 	use_robot = false;
+
+	Gripper gr(limb, use_robot);
+
+
+	EXPECT_EQ(false, gr.is_enabled());
+	EXPECT_EQ(false, gr.is_calibrated());
+	EXPECT_EQ(false, gr.is_ready_to_grip());
+	EXPECT_EQ(false, gr.has_error());
```

Comment:
This is not necessarily your fault, but prefer `EXPECT_FALSE` to `EXPECT_EQ(false, `. If that appeared in `test_robot_interface`, please change that as well.